app/src/main/java/com/endava/myendava/adapters/ChipsAdapter.java

```diff
@@ -97,41 +127,88 @@ private String getTagGroupByPosition(int position) {
 
         TextView tagsGroupTextView;
         ChipGroup chipGroup;
+        TextView addTagClick;
```

Comment:
addTagClick does not show the type of view assigned to this instance ... hence I would suggest a renaming to addTagTextView